bootstrap/src/components/controls/ProjectPicker.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import last from "lodash/last";
+
+import InlineLoading from "../InlineLoading";
+import { useProjectId } from "../../contexts/ProjectId";
+import { useProjectList } from "../../contexts/ProjectList";
+
+const isInList = (projectId, projectList) => {
+    return (
+        projectId &&
+        projectList &&
+        projectList.some(project => last(project.links.self.split("/")) === projectId)
```

Comment:
Detail: `project.links.self.split("/"))` is used twice and it is not really clear what it does. Could you make a function for it?